Scripts/Proxies/ViveProxy.cs

```diff
@@ -10,13 +10,18 @@ sealed class ViveProxy : TwoHandedProxyBase
 #if ENABLE_STEAMVR_INPUT
 		SteamVR_RenderModel m_RightModel;
 		SteamVR_RenderModel m_LeftModel;
+#endif
 
 		public override void Awake()
 		{
 			base.Awake();
 			m_InputToEvents = ObjectUtils.AddComponent<ViveInputToEvents>(gameObject);
+#if !ENABLE_STEAMVR_INPUT
```

Comment:
It's required to prevent a material related exception thrown in TwoHandedProxyBases' Start function